templates/CRM/Contact/Form/Merge.tpl

```diff
@@ -130,9 +130,24 @@
               $row.title|substr:0:5 == "Phone"}
 
             <td>
-              {* @TODO check if this is ever an array or a fileName? *}
+
               {* This is on one long line for address formatting *}
-              {if $row.title|substr:0:7 == "Address"}<span style="white-space: pre" id="main_{$blockName}_{$blockId}">{else}<span id="main_{$blockName}_{$blockId}">{/if}{if !is_array($row.main)}{$row.main}{elseif $row.main.fileName}{$row.main.fileName}{else}{', '|implode:$row.main}{/if}</span>
+              {if $row.title|substr:0:7 == "Address"}
+                <span style="white-space: pre" id="main_{$blockName}_{$blockId}">
+              {else}
+                <span id="main_{$blockName}_{$blockId}">
+              {/if}
+
+              {* @TODO check if this is ever an array or a fileName? *}
+              {if !is_array($row.main)}
+                {$row.main}
+              {elseif $row.main.fileName}
+                {$row.main.fileName}
+              {else}
+                {', '|implode:$row.main}
+              {/if}
+
+              </span>
```

Comment:
As described in the comment, changing the formatting breaks the display of addresses. See the attached screenshot.
![broken-address-format](https://cloud.githubusercontent.com/assets/5212601/20346436/def03726-abf3-11e6-9453-3398eed3348c.PNG)